test/Dialect/FIRRTL/parse-errors.fir

```diff
@@ -1371,55 +1371,55 @@ circuit Foo:
 ;// -----
 FIRRTL version 4.0.0
 circuit Foo:
-  layer A bind:
+  layer A, bind:
   public module Foo:
     output a: Probe<
     ; expected-error @below {{expected probe data type}}
 ;// -----
 FIRRTL version 4.0.0
 circuit Foo:
-  layer A bind:
+  layer A, bind:
```

Comment:
Why this change?

Reply:
Despite the grammar in the spec, there are mixed examples: some with an interleaving comma and some without. I made the call here to require the comma rather than to forbid it.